src/core_cli/Core_CLI.ml

```diff
@@ -658,6 +658,7 @@ let options actions =
       " <file> log debugging info to file" );
     ("-test", Arg.Set test, " (internal) set test context");
     ("-lsp", Arg.Set lsp, " connect to LSP lang server to get type information");
+    ("-raja", Arg.Set Flag_semgrep.raja, " undocumented");
```

Comment:
I know it's an experiment but I think it's still worth putting in some effort to think of a flag name 😅. To this day semgrep still passes `-fast` to `semgrep-core`